std_msgs/Header header
ObstacleDelphiESR[] obstacles